editor/editor_inspector.cpp

```diff
@@ -1082,6 +1160,7 @@ void EditorProperty::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "draw_warning"), "set_draw_warning", "is_draw_warning");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "keying"), "set_keying", "is_keying");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "deletable"), "set_deletable", "is_deletable");
+	ADD_PROPERTY(PropertyInfo(Variant::ARRAY, "config_info"), "set_config_info", "get_config_info");
```

Comment:
Why is this a property?
Is it even useful as readonly? (for custom EditorProperties or something)